etc/visual-studio/libopenthread-ncp-uart.vcxproj

```diff
@@ -73,4 +76,4 @@
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets">
   </ImportGroup>
-</Project>
+</Project>
```

Comment:
Thanks for looking into this.  I'm fine with accepting this PR as is.  Just wanted to make sure we weren't making unnecessary changes.